mediasms.go

```diff
@@ -0,0 +1,114 @@
+package mediasms
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/comvex-jp/mediasms-go/errors"
+	"github.com/comvex-jp/mediasms-go/models"
+	"github.com/comvex-jp/mediasms-go/translations"
+)
+
+// Client struct
+type Client struct {
+	Username string
+	Password string
+	Prefix   string
+}
+
+// resultCode from media4u
+type resultCode struct {
+	Result string `json:"result"`
+}
+
+func createSMSID(prefix, messageID string) string {
+	return prefix + messageID
+}
+
+// SMSURL constant for sms-console
+const SMSURL = "https://www.sms-console.jp/"
```

Comment:
SMS_CONSOLE_URL

Reply:
@anderson-comvex 
`don't use ALL_CAPS in Go names; use CamelCase ` according to the linter.